### PR TITLE
Improve xml version extraction

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Improve xml version extraction - Add exception
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | -
| How to test?      | Try to use xml file without version

### CODE REVIEW

**Review by ga-devfront — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
@M0rgan01 is this the expected result?

image

**Comment by M0rgan01** [comment]
@Quetzacoalt91 I adjusted the wording to have "We couldn't find a PrestaShop version in the .zip file that was uploaded in your local archive. Please try again."

File: classes/Task/Miscellaneous/UpdateConfig.php [hunk 1]
```diff
@@ -97,7 +97,14 @@ public function run(): int
                 return ExitCode::FAIL;
             }
 
-            $xmlVersion = $this->getXmlVersion($fullXmlPath);
+            try {
+                $xmlVersion = $this->extractPrestashopVersionFromXml($fullXmlPath);
+            } catch (Exception $exception) {
+                $this->setErrorFlag();
+                $this->logger->info($this->translator->trans('We couldn\'t find a PrestaShop version in the XML file that was uploaded in your local archive. Please try again'));
```
Comment (Quetzacoalt91) [inline classes/Task/Miscellaneous/UpdateConfig.php:104]: Did you mean `error()`?

```suggestion
                $this->logger->error($this->translator->trans('We couldn\'t find a PrestaShop version in the XML file that was uploaded in your local archive. Please try again'));
```

Comment (M0rgan01) [reply classes/Task/Miscellaneous/UpdateConfig.php:104]: All the "errors" linked to the channel archive are reported in info. Either continue on the same path or we change everything

Comment (M0rgan01) [reply classes/Task/Miscellaneous/UpdateConfig.php:104]: this is returned as a "quickInfo"

Comment (Quetzacoalt91) [reply classes/Task/Miscellaneous/UpdateConfig.php:104]: After discussion, we must leave this as-is to have the result displayed on the interface.

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**